scripts/macos/install-wezterm.sh

```
@@ -1,185 +1,185 @@
-    info "You can now start WezTerm from Applications or run 'open -a WezTerm'"
-    info "Your WezTerm config is linked from: $(readlink "$HOME/.config/wezterm/wezterm.lua")"
-    # Non-interactive: do not prompt or auto-launch applications
-  else
-    error "Installation verification failed"
-    exit 1
-  fi
-}
-
-# Run main function with all arguments
-main "$@"
+#!/usr/bin/env bash
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)